apptools/naming/dynamic_context.py

```diff
@@ -82,7 +82,7 @@ def _is_context(self, name):
         item = self._get_contributed_context_item(name)
         if item != (None, None):
             obj, trait = item
-            result = True == trait.is_context
+            result = trait.is_context is True
```

Comment:
```suggestion
            result = True if trait.is_context is True else False
```

Reply:
I didn't spend much time looking into this but im quite confused by this code.  Why exactly can't we just use `result = traits.is_context`?  I'm assuming it is because `traits.is_context` could take other values besides true or false

Reply:
>  I'm assuming it is because traits.is_context could take other values besides true or false

that's what i assume. not sure what `trait.is_context` is.